src/test/java/seedu/letsgethired/logic/commands/FindCommandTest.java

```diff
@@ -55,7 +62,8 @@ public void equals() {
     @Test
     public void execute_longString_noInternApplicationFound() {
         String expectedMessage = String.format(MESSAGE_INTERN_APPLICATIONS_LISTED_OVERVIEW, 0);
-        CompanyPartialMatchPredicate predicate = new CompanyPartialMatchPredicate("Lorem ipsum 1234");
+        CompanyContainsFieldKeywordsPredicate predicate = new CompanyContainsFieldKeywordsPredicate(
+                Arrays.asList(new Pair<>(PREFIX_COMPANY, "Lorem ipsum 1234")));
```

Comment:
Perhaps we could change `Lorem ipsum 1234` to something else that is clearer to the other developers?

Reply:
Also the test seems to be testing for long String instead of multiple keywords, perhaps we could change the sample test case to `thisisaverylongcompanyname` or something similar?